src/render/OpenGL.cpp

```diff
@@ -89,8 +89,19 @@ GLuint CHyprOpenGLImpl::createProgram(const std::string& vert, const std::string
     GLint ok;
     glGetProgramiv(prog, GL_LINK_STATUS, &ok);
     if (dynamic) {
-        if (ok == GL_FALSE)
+        if (ok == GL_FALSE) {
+            GLint maxLength = 0;
```

Comment:
make a `static` function above to log an error instead of duplicating code

Reply:
Not sure how possible that is since they call different functions (`glGetShaderInfoLog` vs `glGetProgramInfoLog`) and use different error phrasing (`"Error linking program"` vs `"Error compiling shader"`)

Reply:
`bool link`?

Reply:
Tried that, should be good now?